test/unit/math/prim/fun/zeros_vector_test.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stan/math/prim.hpp>
+#include <gtest/gtest.h>
+
+TEST(MathFunctions, zeros_vector) {
+  using Eigen::VectorXd;
+  using stan::math::zeros_vector;
+
```

Comment:
[optional]
This is another one to do in a loop.  Really, so are most of the others.